framework/src/utils/InputParameters.C

```diff
@@ -160,6 +160,18 @@ InputParameters::operator+=(const InputParameters & rhs)
 
 void
 InputParameters::addCoupledVar(const std::string & name, Real value, const std::string & doc_string)
+{
+  // std::vector<VariableName>(1, Moose::stringify(value)),
```

Comment:
remove this line. it looks like copy'n'paste from the other `addCoupledVar` method.